docs/dapp/tools/localnet.md

```diff
@@ -0,0 +1,118 @@
+# Local Development
```

Comment:
I think we need to have the "Localnet" term in the title. So "Localnet Development", "Using Localnet", "Running Localnet" or just "Localnet".